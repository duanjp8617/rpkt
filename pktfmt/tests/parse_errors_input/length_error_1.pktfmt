packet FakeProto {
    header = [
        field_1 = Field {
            bit = 8, 
            gen = false,
        },
        field_2 = Field {
            bit = 8, 
        },
        field_3 = Field {
            bit = 8, 
        }
    ],
    length = [
        header_len =field_1*3,
        payload_len =,
        packet_len=
    ]
}

