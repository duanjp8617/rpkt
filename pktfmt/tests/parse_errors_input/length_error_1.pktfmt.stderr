ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/length_error_1.pktfmt 14:5-18:5
         ~~~~~~~~~~
14 |     length = [
15 |         header_len =field_1*3,
16 |         payload_len =,
17 |         packet_len=
18 |     ]
     ^^^^^
length error 1:
invalid packet length format
